index.html

```diff
@@ -4,10 +4,161 @@
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <h2 class="center-headline">Skills</h2>
+      <article class="intro">
+        <figure class="delet-margin">
+          <img
+            height="200"
+            src="https://images.unsplash.com/photo-1517849845537-4d257902454a?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"
+            alt="Shows a black littel dog wearing a scarf"
+          />
+          <figcaption>I'm very happy!</figcaption>
+        </figure>
+        <div class="skills">
+          <ul>
+            <li>
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+              Accusantium, dolor.
+            </li>
+            <li>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam,
+              velit?
+            </li>
+            <li>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              amet?
+            </li>
+            <li>
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+              Accusantium, dolor.
+            </li>
+            <li>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam,
+              velit?
+            </li>
+            <li>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              amet?
+            </li>
+          </ul>
+        </div>
+      </article>
+
+      <section class="social-cards">
+        <h2 class="center-headline">Social Media</h2>
+        <div class="flex-container">
+          <div class="card">
+            <a href="https://github.com/">
+              <figure>
+                <img
+                  height="200"
+                  width="200"
+                  src="https://play-lh.googleusercontent.com/PCpXdqvUWfCW1mXhH1Y_98yBpgsWxuTSTofy3NGMo9yBTATDyzVkqU580bfSln50bFU"
+                  alt="Shows the Github logo"
+                />
+                <figcaption>Github</figcaption>
+              </figure>
+            </a>
+          </div>
+          <div class="card">
+            <a href="https://codepen.io/">
+              <figure>
+                <img
+                  height="200"
+                  width="200"
+                  src="https://media.licdn.com/dms/image/C560BAQGKtd1slGeUMA/company-logo_200_200/0/1564001544623?e=2147483647&v=beta&t=hxHAT6zqF3yYJLxhRHWSQgWH9y40xhk0kWerZ3LGxI8"
+                  alt="Shows the Github logo"
+                />
+                <figcaption>Codepen</figcaption>
+              </figure>
+            </a>
+          </div>
+          <div class="card">
+            <a href="https://de.linkedin.com/">
+              <figure>
+                <img
+                  height="200"
+                  width="200"
+                  src="https://www.svgrepo.com/show/922/linkedin.svg"
+                  alt="Shows the Github logo"
```

Comment:
```suggestion
                  alt="LinkedIn logo"
```
Copy Paste mistakes ;)